src/Mapper/ProgrammesDbToDomain/MapperFactory.php

```diff
@@ -4,8 +4,29 @@
 
 class MapperFactory
 {
+    /** @var array */
     protected $instances = [];
 
+    /** @var array */
+    private $options = [
+        // In Clifton the MasterBrand is the one associated with the current
```

Comment:
PPS is just a system agnostic package added via Composer. Should such a package mention specific use cases like "Clifton" or "Frontend"?